test/CMakeLists.txt

```diff
@@ -213,7 +213,7 @@ else()
 endif()
 
 if (MIOPEN_NO_GPU)
-    set(SKIP_ALL_EXCEPT_TESTS test_include_inliner test_kernel_build_params test_lstm test_lstm_dropout
+    set(SKIP_ALL_EXCEPT_TESTS test_include_inliner test_kernel_build_params
```

Comment:
This change seems to be out of the scope of this PR.

Reply:
@atamazov , you are correct. This is for the following PR. Since it's minor, I put it here too to avoid merge conflict. Would you suggest to revert it? Thanks

https://github.com/ROCmSoftwarePlatform/MIOpen/issues/1628

Reply:
Yes, please revert.

Reply:
Reverted. Thanks :)